Aztec/Classes/TextKit/TextView.swift

```diff
@@ -1220,11 +1210,10 @@ open class TextView: UITextView {
     /// - Returns: the attachment object that can be used for further calls
     ///
     @discardableResult
-    open func replaceRangeWithCommentAttachment(_ range: NSRange, text: String) -> CommentAttachment {
-        let attachment = storage.replaceRangeWithCommentAttachment(range, text: text, attributes: typingAttributes)
-
-        selectedRange = NSMakeRange(range.location + NSAttributedString.lengthOfTextAttachment, 0)
-        delegate?.textViewDidChange?(self)
+    open func insertComment(at range: NSRange, text: String) -> CommentAttachment {
+        let attachment = CommentAttachment()
+        attachment.text = text
+        insert(attachment: attachment, at: range)
```

Comment:
We should roll back this name change.

We're still replacing a range with new contents, as opposed to inserting without deleting anything that's in there.